src/commands/lambda/flare.ts

```diff
@@ -371,16 +422,28 @@ export const createDirectories = (
  * Gets the LOG_STREAM_COUNT latest log stream names, sorted by last event time
  * @param cwlClient CloudWatch Logs client
  * @param logGroupName name of the log group
+ * @param startMillis start time in milliseconds or 0 if no start time is specified
+ * @param endMillis end time in milliseconds or 0 if no end time is specified
  * @returns an array of the last LOG_STREAM_COUNT log stream names or an empty array if no log streams are found
  * @throws Error if the log streams cannot be retrieved
  */
-export const getLogStreamNames = async (cwlClient: CloudWatchLogsClient, logGroupName: string) => {
-  const command = new DescribeLogStreamsCommand({
+export const getLogStreamNames = async (
+  cwlClient: CloudWatchLogsClient,
+  logGroupName: string,
+  startMillis: number,
+  endMillis: number
+) => {
+  const config = {
     logGroupName,
-    limit: LOG_STREAM_COUNT,
     descending: true,
     orderBy: OrderBy.LastEventTime,
-  })
+    limit: DEFAULT_LOG_STREAMS,
+  }
+  const rangeSpecified = startMillis !== 0 || endMillis !== 0
```

Comment:
Why `||`? Isn't it guaranteed that we only run this when both of them are set?

Reply:
It's possible that the user intentionally set `startMillis` to 0.
For example, if they wanted to get all log streams, they might do: `--start 0 --end 9999999999999`.
In this case, `||` is required to detect that a range was specified.

Reply:
Makes sense, this would've helped when this values are `undefined` as mentioned on a thread above.